__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const db = require("../db/connection");
+
+beforeEach(() => {
+  return seed(data);
+});
+afterAll(() => {
+  db.end();
+});
+describe("express server", () => {
+  test("404, endpoint does not exist", () => {
+    return request(app)
+      .get("/api/torpics")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body).toEqual({ msg: "endpoint does not exist" });
+      });
```

Comment:
Just as a note for reference, this test may be included in this PR but it is a generic test for a 404 and is not specific to this endpoint.